client/css/jsonedit.css

```diff
@@ -351,6 +351,10 @@ body.jsonEdit.jeZoomOut #topSurface {
   color: orange;
 }
 
+.jeLogTime {
+  color: #444;
+}
```

Comment:
My old eyes cannot see this. Perhaps in yellow, enclosed in parentheses?

Reply:
Better?